.gitignore

```diff
@@ -1,5 +1,7 @@
+.puli/
 build/
 vendor/
 composer.lock
 phpspec.yml
 phpunit.xml
+puli.json
```

Comment:
I see. Are those packages loaded into the application where the package is installed afterwards? Because I probably don't want integration testing binding in the application where I use the package.